core-bundle/src/Controller/ContentElement/PlayerController.php

```diff
@@ -113,11 +121,42 @@ function (FilesystemItem $item) use (&$captions, $range): HtmlAttributes {
             $sourceFiles,
         );
 
+        $tracks = [];
+
+        $setDefault = false;
+
+        if ([] !== $subtitlesFiles) {
+            foreach ($subtitlesFiles as $file) {
+                $subtitles = $file->getExtraMetadata()['subtitles'] ?? null;
+                $label = ($file->getExtraMetadata()['metadata'] ?? null)?->getDefault()?->getTitle();
```

Comment:
```suggestion
                $subtitles = $file->getExtraMetadata()->getSubtitles();
                $label = $file->getExtraMetadata()->getLocalized()?->getDefault()?->getTitle();
```
This needs #7626 to be merged upstream first. Then the `getSubtitles()` method can be added analogous to these: https://github.com/contao/contao/blob/ee27c852e2ea8dc0907e02e1e99ea346a80d57f4/core-bundle/src/Filesystem/ExtraMetadata.php#L56-L76

Reply:
Should somehow add a ToDo for that one since this would be unrelated to this PR but this looks way cleaner <3

Reply:
IMHO we should merge #7626 first and then adjust this PR. It cannot be done the other way round, because #7626 is targeting 5.3. Alternatively, we can merge this PR as is and create a followup PR. 

@leofeyer Any preferences?

Reply:
#7626 has been merged 👍

Reply:
@m-vo @leofeyer done in 621127de1aa16f3ddddd51cb8010f9cb81bdf000 :)